projects/commudle-admin/src/app/feature-modules/notifications/components/notifications-page/notifications-page.component.scss

```diff
@@ -0,0 +1,80 @@
+@import './projects/commudle-admin/src/assets/styles/variables';
+
+.container {
+  max-width: 768px;
```

Comment:
any reason?

Reply:
remove max-width, add to clickup for one page layout